### DISCUSSION TITLE
Meeting Minutes May 15, 2023

### DISCUSSION THREAD

**Discussion by jialincheoh** [ironhacks/ironhacks-app#171] [Ideas]
- Discussion on Paper Progress 
- Follow up with CLA Faculty after the workshop 
- Wrapping up Fall 2022 hack
- Cleaning up of IronHacks repositories

**Comment by ManthanKeim** [comment]
https://docs.google.com/document/d/17zpV5uORD5-O5NOzIcb79H90WSfWkuNx4q588k7HVUo/edit
